ideas/313-sticker-market.md

```diff
@@ -0,0 +1,71 @@
+Sticker Market 
+===
+
+## Summary and Goals
+
+Stickers are a must-have for any messenger, particularly with Asian markets in mind.
+
+The goal of the Status sticker market is two-fold:
+
+1) Allow users to buy and use stickers in chat.
+2) Allow creators to sell stickers permissionlessly through a curated marketplace.
+
+For the initial scope, the focus is on use case #1.
+
+We can commission a set of stickers to populate the marketplace starting out.
+
+## Contributors
+
+- Julien
+- Andrey
+- Nastya
+- Smart contract dev needed (probably Ricardo)
+- Designer needed
+
+(+ Janitor swarm) 
+
+## Communication
+
+`status channel`: #313-sticker-market
+
+`sync schedule`: biweekly sprint planning, ad hoc meetings as needed
+
+`Meeting notes:` https://www.google.com/url?q=https%3A%2F%2Fnotes.status.im%2FwyjNCxeURu6lYSwWGh3biw&sa=D&ust=1545051619127000&usg=AFQjCNGFgvurpaS4HBPB0Y3s5pR5HLiQWA
+
+## Research 
+
+`Timing`: wrap on 21/12
+
+`References`: LINE & Kakao are two of the best sticker markets. Looking at these two alone can give ample guidance on great sticker market UX. 
```

Comment:
Telegram as well, though no marketplace. Would be interesting to understand interplay between free and paid, as well as things like giphy extensions / memes, and what impact this has for likelihood of people paying. What is the value add?